src/GUI/RevEng.Core.50/CommentCSharpEntityTypeGenerator.cs

```diff
@@ -38,12 +39,14 @@ public class CommentCSharpEntityTypeGenerator : ICSharpEntityTypeGenerator
         /// </summary>
         public CommentCSharpEntityTypeGenerator(
             [NotNull] IAnnotationCodeGenerator annotationCodeGenerator,
-            [NotNull] ICSharpHelper cSharpHelper)
+            [NotNull] ICSharpHelper cSharpHelper,
+            bool nullableReferences)
```

Comment:
Note that on the EF Core side we hope to detect whether this is enabled at the project level, so ideally we wouldn't need to emit anything. But this is fine too (and much easier obviously).

Reply:
Sure, but you would still need to pass that information to the tooling, which knows nothing about the project.